python/openvino_tokenizers/cli.py

```diff
@@ -104,6 +111,17 @@ def get_parser() -> ArgumentParser:
             "Not supported for Sentencepiece-based tokenizers."
         ),
     )
+    parser.add_argument(
+        "--max_length",
+        "--max-length",
+        required=False,
+        type=check_positive_int,
+        help=(
+            "Set max_length to the tokenizer for truncation operation. "
+            "Tokenizer won't produce output longer than max_length. "
+            "The value will be replaced by the max_padding option if set."
```

Comment:
can we also set it as upper bound for output partial shape ?

Reply:
CombineSegments has `max_length` as input but produces a ragged tensor and cannot pass a result tensor dimensions. RaggedToDense operation should set the max_length of the output tensor, but doesn't have direct access to the `max_length` parameter. We can try to access `max_length` constant node and set the upper bound there.